Doc/glossary.rst

```diff
@@ -1118,7 +1118,7 @@ Glossary
    universal newlines
       A manner of interpreting text streams in which all of the following are
       recognized as ending a line: the Unix end-of-line convention ``'\n'``,
-      the Windows convention ``'\r\n'``, and the old Macintosh convention
+      the Windows convention ``'\r\n'``, and the old Mac convention
```

Comment:
I'd prefer to keep Macintosh here. This is referring to MacOS 9 and earlier, when Macintosh was commonly used to describe the system.